simapp/export.go

```diff
@@ -41,9 +43,25 @@ func (app *SimApp) ExportAppStateAndValidators(forZeroHeight bool, jailAllowedAd
 	}
 
 	validators, err := staking.WriteValidators(ctx, app.StakingKeeper)
+	cmtValidators := []cmttypes.GenesisValidator{}
```

Comment:
We should add a note for this in the UGPRADING.md > Simapp.
As well in simapp/CHANGELOG.md

Reply:
Shall we even got one step further, and put `GenesisValidator` under `types`. Update ExportGenesis, genutil etc, to use the SDK `GenesisValidator`, and only do the conversion to comet pubkeys when we call `ToGenesisDoc` ? (https://github.com/cosmos/cosmos-sdk/blob/eeeb5b86b5879c25d72f025351f4c6f9a2ff75e0/x/genutil/types/genesis.go#L147).

That way we really pushed comet at the edge, as currently the whole genesis logic still requires comet pubkeys type then.

Reply:
If we do that, we may need to add a genesis migration in gentutil.
Doesn't have to be done in this already big PR, but that'd be an interesting follow-up (maybe let's add a todo?)